### PR TITLE
Fix proportion OverusedFn overused logic

### PR DESCRIPTION
**Pull request by zen-xu**
fix-bug: #1115

Signed-off-by: ZhengYu, Xu <zen-xu@outlook.com>

### CODE REVIEW

**Comment by zen-xu** [comment]
Note: This pr need #1037 fix first

File: pkg/scheduler/plugins/proportion/proportion.go [hunk 1]
```diff
@@ -242,7 +242,7 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 		queue := obj.(*api.QueueInfo)
 		attr := pp.queueOpts[queue.UID]
 
-		overused := !attr.allocated.LessEqual(attr.deserved)
+		overused := !attr.allocated.Less(attr.deserved)
```
Comment (Thor-wl) [inline pkg/scheduler/plugins/proportion/proportion.go:245]: Please make CI happy

Comment (zen-xu) [reply pkg/scheduler/plugins/proportion/proportion.go:245]: #1036 lead the CI failed, so need to fix it first.

**Comment by Thor-wl** [comment]
Perhaps you need to go format the code

**Pull request closed without merging**